### ISSUE TITLE
full screen problem

### ISSUE THREAD

**Issue by gabriel-tandil** [jitsi/jitsi-meet#15410]
### What happened?

Occasionally, quite often, when I go back to the application, the drawing area of the application also occupies the android button bar and is covered, which makes it very difficult to switch the mute button or hang up the call. I add an image to clarify.

![imagen](https://github.com/user-attachments/assets/79b99c44-0902-49ca-8489-9197b293215e)

### Platform

- [ ] Chrome (or Chromium based)
- [ ] Firefox
- [ ] Safari
- [ ] Other desktop browser
- [ ] Android browser
- [ ] iOS browser
- [ ] Electron app
- [X] Android mobile app
- [ ] iOS mobile app
- [ ] Custom app using a mobile SDK

### Browser / app / sdk version

24.5.8

### Reproducibility

- [ ] The problem is reproducible on meet.jit.si

### More details?

I still can't figure out the exact steps that cause this error. It may be related to the screen lock.

**Comment by saghul** [comment]
Hum, we never released a 24.5.8. Where did you download the app from? Can you test 24.6.0?

**Comment by gabriel-tandil** [comment]
I always download it from fdroid, I will try to update from this repository releases directly.

**Comment by gabriel-tandil** [comment]
I tested it with the latest version in the playstore and I have the same behavior, after the screen lock part of the screen is hidden behind the android navigation bar.
Can I support you with any log or any test that you indicate me?

**Comment by saghul** [comment]
Can you provide details steps for reproducing the problem?

**Comment by gabriel-tandil** [comment]
Of course, I'm sharing a video where you can see that after locking the screen and returning to the app, the buttons to hang up the call or mute are no longer displayed correctly.

https://github.com/user-attachments/assets/8bb1fae4-dc27-48aa-be3e-9a3fdaa03a1b

**Comment by saghul** [comment]
This is helpful, thank you! We are winding down for the end of the year, we'll take a look after the break!

/cc @Calinteodor

**Comment by saghul** [comment]
No news, sorry, it fell through the cracks. @Calinteodor can you PTAL when you get a chance?

**Comment by Calinteodor** [comment]
Will take a look, sure.
